### DISCUSSION TITLE
No accepted nightlies for 12 days

### DISCUSSION THREAD

**Discussion by fortinj66** [okd-project/okd#2064] [General]
Is anyone looking to see why nightlies have been failing for the last 12 days or so?

![image](https://github.com/user-attachments/assets/d2d7d4e8-af56-4033-a00d-dce88a190b6c)

**Comment by Prashanth684** [comment]
Thanks for reporting the bug! We have been looking into this for a while now: https://issues.redhat.com/browse/OKD-231 and there is no clear indication as to what is broken with that one test. That said, since we have started promoting 4.17 releases to stable, we will no longer be releasing/promoting 4.16 builds.

**Reply by fortinj66** [reply]
Ok, what will the official 4.16 release be then as that will be needed to upgrade from 4.15.

**Reply by tyronewilsonfh** [reply]
The target release from 4.15.0-0.okd-2024-03-10-010116 is currently 4.16.0-okd-scos.1, I tested this yesterday and we will update the docs with the upgrade process
